### PR TITLE
refresh the network-firewalls-stonesoft documentation

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 1]
```diff
@@ -2,36 +2,753 @@
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Stonesoft-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                | Modèle de service                              | Description                                    | Découverte |
+|:---------------------|:-----------------------------------------------|:-----------------------------------------------|:----------:|
+| Cluster-Load         | Net-Stonesoft-Cluster-Load-SNMP-custom         | Contrôle de la charge consommée par le cluster |            |
+| Cluster-State        | Net-Stonesoft-Cluster-State-SNMP-custom        | Contrôle l'état du cluster                     |            |
+| Disk-Global          | Net-Stonesoft-Disk-Global-SNMP-custom          | Contrôle du taux d'utilization des disques     |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:39]:
```suggestion
| Disk-Global          | Net-Stonesoft-Disk-Global-SNMP-custom          | Contrôle du taux d'utilisation des disques     |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 2]
```diff
@@ -2,36 +2,753 @@
...
+| Disk-Name            | Net-Stonesoft-Disk-Name-SNMP-custom            | Contrôle du taux d'utilization des disques     |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:40]:
```suggestion
| Disk-Name            | Net-Stonesoft-Disk-Name-SNMP-custom            | Contrôle du taux d'utilisation des disques     |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 3]
```diff
@@ -2,36 +2,753 @@
...
+| *interface_name*#interface.traffic.out.bitspersecond | b/s   |
+
+> Concerne les modèles de service suivants : Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:159]: missing prerequisites

File: pp/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 4]
```diff
@@ -2,36 +2,752 @@
...
+
+> Applies to the following service templates: Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-This chapter describes the prerequisites installation needed by plugins to run.
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:158]: missing prerequisites

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 5]
```diff
@@ -2,36 +2,753 @@
...
+to the manufacturer official documentation BUT you should try to be as complete\n\
+as possible here as it will save time to everybody.*
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:161]:
```suggestion
### Configuration SNMP

Le service SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.

### Flux réseau

La communication doit être possible sur le port UDP 161 depuis le collecteur Centreon vers la ressource supervisée.
```

File: pp/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md [hunk 6]
```diff
@@ -2,36 +2,752 @@
...
+to the manufacturer official documentation BUT you should try to be as complete\n\
+as possible here as it will save time to everybody.*
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-firewalls-stonesoft.md:160]:
```suggestion
### SNMP Configuration

The SNMP service must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

### Network flow

The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**